app/models/complaint.rb

```diff
@@ -0,0 +1,97 @@
+class Complaint
+  attr_reader :id, :attributes
+
+  EVENT_FIELDS = %w[
+    issueLastUpdated
+    creationDate
+    closedDate
+    reopenedDate
+    initialContactDate
+  ].freeze
+
+  EVENT_LABELS = {
+    issueLastUpdated: "Updated",
+    creationDate: "Created in HSES",
+    closedDate: "Closed",
+    reopenedDate: "Reopened",
+    initialContactDate: "Initial contact from complaint"
+  }.with_indifferent_access.freeze
+
+  def initialize(hses_complaint)
+    @id = hses_complaint[:id]
+    @attributes = hses_complaint[:attributes].with_indifferent_access
```

Comment:
i think it might be nice to make an events list that is comprised of event objects?